src/css/controls/imports/playback-label.less

```diff
@@ -1,8 +1,12 @@
 @import "../../shared-imports/vars.less";
 
+.jw-idle-icon-text {
+    display: none;
+}
+
 .jw-state-idle {
     .jw-icon-display {
-        &.jw-ab-idle-label {
+        &.jw-idle-label {
```

Comment:
Same here - if we move the default styling for `.jw-idle-label` to outside `.jw-state-idle` its easier to re-use for other player states.